library/build.gradle

```diff
@@ -25,6 +25,10 @@ android {
   }
 }
 
+group = "com.github.prolificinteractive"
+version = android.defaultConfig.versionName
+archivesBaseName = 'material-calendarview'
+
```

Comment:
This will automatically name the artifact that is produced to material-calendarview-2.0.1.aar if the version is set to 2.0.1.  This will help to automate publishing to maven repositories and this helps with enabling 1 step deployment to the local maven repository using "./gradlew install" so that it matches what is in the public maven repo.  This allows consumers to make a change then quickly deploy to their local maven repo and test the change by just bumping their dependency in their build.gradle file and having mavenLocal() in their repository list.

I would be glad to help setup the jcenter repo and enable publishing to it if you would like.  I felt like that was a later pull request should you want that change too.